server/src/internalClusterTest/java/org/elasticsearch/index/store/CorruptedFileIT.java

```diff
@@ -616,6 +616,7 @@ public void testCorruptFileThenSnapshotAndRestore() throws InterruptedException,
      * nodes, so that replica won't be sneaky and allocated on a node that doesn't have a corrupted
      * replica.
      */
+    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/86429")
```

Comment:
I believe this is outdated, test was fixed in https://github.com/elastic/elasticsearch/pull/91272

Reply:
Good point, addressed in 4e4a7b6aa14d5126fc562de6a1d25869aa53acbc.